lib/manager/helmv3/artifacts.spec.ts

```diff
@@ -59,93 +65,96 @@ describe('manager/helmv3/artifacts', () => {
     ).toBeNull();
   });
   it('returns null if unchanged', async () => {
-    fs.readFile.mockResolvedValueOnce('Current Chart.lock' as any);
+    fs.readFile.mockResolvedValueOnce(ociLockFile1 as any);
```

Comment:
```suggestion
    fs.readFile.mockResolvedValueOnce(ociLockFile1);
```
I think the any casts are no longer needed.

Please remove them all where possible

Reply:
Typescript fails to compile, if I remove this as it will not choose the correct signature of `readFile`.